doc/api.md

```diff
@@ -195,27 +195,27 @@ Array of pe supported versions
 
 ## Schemas
 
-### minecraft-data.schema.biomes
```

Comment:
That would require to release a major version. Why do we need to change that ?

Reply:
[The API is wrong](https://github.com/PrismarineJS/node-minecraft-data/blob/8abf8ed0ff9684bd71ecc1a9b980d94540f16549/index.js#L71), I was just fixing it.